pyproject.toml

```
@@ -1,21 +1,24 @@
 [build-system]
-requires = ["hatchling", "editables"]
-build-backend = "hatchling.build"
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
 
 [project]
-name = "pytest-mpi"
-version = "0.1"
+name = "mpi-pytest"
+# <year.month.patch>
+version = "2025.2.0"
 dependencies = ["mpi4py", "pytest"]
 authors = [
   { name="Connor Ward", email="c.ward20@imperial.ac.uk" },
+  { name="Jack Betteridge" },
```

```suggestion
  { name="Jack Betteridge" , email="j.betteridge@imperial.ac.uk" },
```
You can use this one for the time being!